index.html

```diff
@@ -677,7 +677,7 @@
         <div class="col-sm-9 col-xs-12">
           <p>The Workshop Venue is set in the former Imperial post office in Französische Strasse, right in Berlin Mitte, just meters away from Friedrichstrasse and the Gendarmenmarkt. Lunch will be catered at the venue and of course we'll have coffee and drinks available during the day.</p>
           <p>Also, as has become a tradition already, simplabs invites all Workshop Attendees to a shared dinner on the night of the first day 🍔🍹.</p>
-          <p><b>Französische Str. 12 10117 Berlin Germany</b></p>
+          <b><p>Französische Str. 12<br> 10117 Berlin<br> Germany</p></b>
```

Comment:
I think it would be a good idea to reuse this: https://github.com/simplabs/simplabs.github.io/blob/master/contact/index.html#L59

I don't think `<p>` inside of `<b>` is allowed BTW.